### ISSUE TITLE
database user is not connecting correctly 

### ISSUE THREAD

**Issue by mohamedahmed01** [zsoerenm/magento2-docker#3]
logging is showing :
db_1 | 2019-07-01 13:48:43 140675136968448 [Warning] Access denied for user 'root'@'localhost' (using password: NO)

**Comment by araujoantonio** [comment]
Did you manage to solve this problem?

**Comment by zsoerenm** [comment]
Yes, I also see those warnings and I wonder what is triggering it.
The connection to the database works, though.
Please check out the latest update.

**Comment by webbrain-one** [comment]
Thanks for reporting! I've opened a pull request that fixes the database user connection issue (the “Access denied for user ‘root’” warning). You can find the changes in [PR #48](https://github.com/zsoerenm/magento2-docker/pull/48).
